skcosmo/preprocessing/KernelCentering.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+import sklearn as sk
+from sklearn.base import TransformerMixin, BaseEstimator
+
+
+class KernelCentering(TransformerMixin, BaseEstimator):
+    """Kernel centering method, similar to KernelCenterer,
+    but with additional parameters, relative to which centering
```

Comment:
This needs more descriptive doc-strings. What are the additional parameters? How does this differ?